src/huggingface_hub/hf_api.py

```diff
@@ -1431,7 +1427,7 @@ def list_repo_files(
         )
         return [f.rfilename for f in repo_info.siblings]
 
-    @_deprecate_positional_args
+    @_deprecate_positional_args(version="0.8")
```

Comment:
This was the default value. I either suggest to increase to version to "0.10" or "0.12" or just to remove the deprecation and consider that other libraries made the change already.

IDK the history of this deprecation but my opinion would be to wait for v0.12 and remove all positional args at the same time.